public/mockServiceWorker.js

```diff
@@ -2,22 +2,21 @@
 /* tslint:disable */
 
 /**
- * Mock Service Worker (0.36.4).
+ * Mock Service Worker (0.44.2).
```

Comment:
CLI command execution updated mock service worker.
(I may need a Github action to execute CLI automatically when `msw` is updated in the future.)